protocol-units/bridge/contracts/test/NativeBridge.t.sol

```diff
@@ -60,22 +67,93 @@ contract NativeBridgeTest is Test {
 
         bytes32 bridgeTransferId = nativeBridge.initiateBridgeTransfer(_recipient, _amount);
 
-        (address originator, bytes32 recipient_, uint256 amount, uint256 nonce) =
-            nativeBridge.outgoingBridgeTransfers(bridgeTransferId);
+        (bytes32 bridgeTransferId_, address originator, bytes32 recipient_, uint256 amount) =
+            nativeBridge.noncesToOutboundTransfers(1);
 
         assertEq(originator, _originator);
         assertEq(recipient_, _recipient);
         assertEq(amount, _amount);
-        assertEq(nonce, 1);
+        assertEq(bridgeTransferId_, bridgeTransferId);
+        vm.stopPrank();
+    }
+
+    function testInboundRateLimitFuzz(address receiver, uint256 _amount, uint256 _denominator) public {
+        uint256 insuranceBalance = moveToken.balanceOf(insuranceFund);
```

Comment:
This needs comments. But, it also isn't testing the accumulation against the rate limit.

Reply:
@Primata leave the Fuzz test in but can you add another case to test the accumulation against the rate limit.

Reply:
@Primata this still isn't really testing accumulation very thoroughly.